src/__tests__/render.test.tsx

```diff
@@ -285,3 +285,17 @@ test('RenderAPI type', () => {
   render(<Banana />) as RenderAPI;
   expect(true).toBeTruthy();
 });
+
+test('should throw when rendering a string outside a text component', () => {
+  expect(() => render(<View>hello</View>)).toThrowError(
+    'Text strings must be rendered within a host Text component.'
+  );
+});
+
+test('should throw an error when rerendering with text outside of Text component', () => {
```

Comment:
Would you mind adding a test to check that `<Text><>My test</></Text>` doesn't throw?

Reply:
The validation didn't work when the child was rendered in a fragment, eg <View><>0</></View>, I fixed it and added another test case